README.md

```diff
@@ -1,7 +1,7 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
-Portfolio
+Trail Portfolio Project
 
 > Description the project.
 This a portfolio trials project.
```

Comment:
Please consider adding a professional title and description to your ReadMe.md file,because having a unique and descriptive title for your webpage meets the best accessibility requirements and best SEO standards. that way the screen readers will easily read the title of your webpage and it will have good performance in search engine results.

Reply:
This is a great review, however, be careful because the README doesn't interact with the webpage!